### PR TITLE
Fix #3130: load compendium title

### LINKED ISSUE #3130: Procedures Load Compendium lost title
### Describe the bug

Procedures Load Compendium lost the title.

### To Reproduce

1. Choose the procedures 
2. Click Load Compendium

image

### Client configuration

- Browser: Firefox 73.0.1（64 bit）
- Operating system: Windows 10

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3130 

#### Short description of what this resolves:
Add load compendium title

image

### CODE REVIEW

File: interface/orders/load_compendium.php [hunk 1]
```diff
@@ -81,11 +81,18 @@ function getLabID($npi)
 
 <head>
 <?php Header::setupHeader(); ?>
-<title><?php echo xlt('Load Lab Configuration'); ?></title>
+<title><?php echo xlt('Load Compendium'); ?></title>
```
Comment (bradymiller) [inline interface/orders/load_compendium.php]: Would change this to:
Load Lab Compendium

File: interface/orders/load_compendium.php [hunk 2]
```diff
@@ -81,11 +81,18 @@ function getLabID($npi)
...
 </head>
 
 <body class="body_top text-center">
 <div class="container">
+    <div class="row">
+        <div class="col-12">
+            <div class="page-header text-left">
+                <h2><?php echo xlt('Load Compendium'); ?></h2>
```
Comment (bradymiller) [inline interface/orders/load_compendium.php]: Would change this to:
Load Lab Compendium

File: interface/orders/load_compendium.php (hunk 1 above)
Comment (bradymiller) [reply interface/orders/load_compendium.php]: Actually, this will make the tab to big. Leave this as is, but just change the page title below.

**Comment by stu01509** [comment]
Hi @bradymiller 

I finished modifying, Please review it again.

image

**Comment by stephenwaite** [comment]
thanks for the fix @stu01509 , bringing it in now

**Pull request merged**